src/pages/verify/guides/prefilled-fields.mdx

```diff
@@ -11,20 +11,20 @@ export default Layout;
 
 In some login flows, user-specific data other than actual login details are needed to complete the authentication. 
 
-Examples include the CPR number in Denmark for personal NemID and SSN ("personnummer") in Sweden for BankID on another device.
+Examples include the CPR number in Denmark for personal MitID and SSN ("personnummer") in Sweden for BankID on another device.
 
 By default the user is prompted for this data when needed, but if you would like to avoid these dialogs, you can specify the values up-front in the request for authentication, via the `login_hint` query parameter. This works for both OpenID Connect and WS-Federation.
 
 The actual support for controlling the input data varies by eID method:
 
-1. Danish NemID: Use `login_hint=sub:<CPR>` where `<CPR>` has the format `DDMMYYXXXX`
+1. Danish MitID: Use `login_hint=sub:<CPR>` where `<CPR>` has the format `DDMMYYXXXX`
```

Comment:
Don't believe this is supported for MitID, so might just want to remove this all together, right @sgryt?

Reply:
@mickhansen It is supported for MitID (https://github.com/GreanTech/identityprovider/blob/1d4b7ca8c4baaee454a37eafe38b904e78c02a63/TokenIssuerApi/DanishMitId.fs#L2239C1-L2246C1)